rilldata/rill
# Remove embedded sources from Source UI

## Checklist

## Summary
#### Issue addressed: 
See the discussion at [the top of this Notion doc](https://www.notion.so/rilldata/Refine-the-source-experience-658e67c4039440d89b39385091d90e4e)

#### Details:
This PR removes embedded sources from the Source UI. It retains the embedded source affordances in the Model UI.

## Steps to Verify
1. Use an embedded source in a model
2. Verify that the embedded sources affordances are still present in the Model UI:
    i. References in the Model Inspector
    ii. Ability to cite the model in downstream models
